src/index.ts

```diff
@@ -14,7 +14,8 @@ const cli = meow(
     --tinyIntAsBoolean, -b    Treat TinyInt as Boolean
     --binaryAsBuffer, -B      Treat Binary as Buffer
     --nullAsUndefined, -u     Treat null as undefined   
-    --nullPlusUndefined, -p   Treat as null and undefined   
+    --nullPlusUndefined, -p   Treat as null and undefined
+    --types                   Output types instead of interfaces
```

Comment:
I think it would be better with 
`--outputAsTypes, -t `

Reply:
Or just skip this part and just use types instead of interfaces. Not sure why anyone would care if it's one or the other

Reply:
it actually can make a difference. As I understand a `type` allows keys that are not specified (indexable) while `interface` does not, so `interface` is more strict